assets/css/stripeStyle.css

```diff
@@ -269,3 +265,20 @@ table tr {
 .summary h2 {
 	color: #304ffe;
 }
+
+.error-message-container {
+	width: calc(100% - 11px);
+    background-color: #fff;
+	border: 1px solid #ccd0d4;
+    border-left-width: 4px;
+    box-shadow: 0 1px 1px rgba(0,0,0,.04);
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5894153)